File: C#/bytebank/bytebank/Titular/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Threading.Tasks;

namespace bytebank.Titular
{
    public class Cliente
    {
        private string nome;
        private string cpf;
        private string profissao;   
        public string Nome
        {
            get { return nome; }
            set { nome = value; }
        } //get e set do nome
        public string Cpf
        {
            get { return cpf; }
            set { cpf = value; }
        }//get e set do cpf
        public string Profissao
        {
            get { return profissao; }
            set { profissao = value; }
        }//get e set do profissao
        public Cliente(string nome, string cpf, string profissao)
        {
            this.Nome = nome;
            this.Cpf = cpf;
            this.Profissao = profissao; ;
        }//Contrutor Cliente
        public override string ToString()
        {
            return "********************" + "\nTitular: " + nome + "\nCPF: " + cpf +
               "\nProfissão: " + profissao;
        }//Override da ToString
    }
}

File: JAVA/java-io/src/br/com/alura/java/io/teste/Cliente.java
package br.com.alura.java.io.teste;

import java.io.Serializable;

public class Cliente implements Serializable{

	private static final long serialVersionUID = 6304810090140005569L;
	
	private String nome;
	private String cpf;
	private String profissao;

	public Cliente(String nome, String cpf, String profissao) {
		this.nome = nome;
		this.cpf = cpf;
		this.profissao = profissao;
	}

	public String getNome() {
		return this.nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getCpf() {
		return this.cpf;
	}

	public void setCpf(String cpf) {
		this.cpf = cpf;
	}

	public String getProfissao() {
		return this.profissao;
	}

	public void setProfissao(String profissao) {
		this.profissao = profissao;
	}

	@Override
	public String toString() {
		return "*****CLIENTE****"+
				"\nTitular: "+this.nome+
				"\nCPF: " + this.cpf +
				"\nProfissão: " +this.profissao+
				"\n*******************";
	}
}
